extensions/pkg/webhook/controlplane/test/matchers.go

```diff
@@ -48,7 +50,9 @@ func (m *containElementWithPrefixContainingMatcher) Match(actual interface{}) (s
 		return false, nil
 	}
 	values := strings.Split(strings.TrimPrefix(items[i], m.prefix), m.sep)
-	j := extensionswebhook.StringIndex(values, m.value)
+	j := slices.IndexFunc(values, func(s string) bool {
+		return s == m.value
+	})
```

Comment:
The same can be done for the other occurrences

```suggestion
	j := slices.Index(values, value)
```